manim/mobject/svg/tex_mobject.py

```diff
@@ -61,6 +71,9 @@ def __init__(self, tex_string, **kwargs):
         if self.organize_left_to_right:
             self.organize_submobjects_left_to_right()
 
+    def __repr__(self):
+        return f"{type(self).__name__}({repr(self.tex_string)})"
```

Comment:
Question (non-blocking): what is the difference between `type(self).__name__` and `self.__class__.__name__`?

Reply:
The documentation says that they are generally the same (https://docs.python.org/3/library/functions.html#type).

I usually try to avoid accessing underscore methods directly, but I am fine with changing if you prefer `self.__class__.__name__`.

Reply:
Nah, that's fine I was just curious